components/Marketing/Trial.js

```diff
@@ -0,0 +1,211 @@
+import SwitchBoard from '../Auth/SwitchBoard'
+import withMe from '../../lib/apollo/withMe'
+import withT from '../../lib/withT'
+import { TRIAL_CAMPAIGN } from '../../lib/constants'
+
+import { Button, Field, InlineSpinner, Interaction, colors } from '@project-r/styleguide'
+
+const { H1, P } = Interaction
+
+const styles = {
+  errorMessages: css({
+    color: colors.error,
+    marginTop: 40
+  }),
+  switchBoard: css({
+    marginTop: 40
+  }),
+  button: css({
+    marginTop: 40,
+    width: 180,
+    textAlign: 'center'
+  })
+}
+
+const REQUIRED_CONSENTS = ['PRIVACY', 'TOS']
+
+const REQUEST_ACCESS = gql`
+  mutation requestAccess($campaignId: ID!) {
+    requestAccess(campaignId: $campaignId) {
+      id
+      endAt
+    }
+  }
+`
+
+const toOnboarding = () => {
+  window.location = format({ pathname: '/einrichten', query: { context: 'trial' } })
+}
+
+const Trial = (props) => {
+  const { me, t } = props
+
+  const [consents, setConsents] = useState()
+  const [email, setEmail] = useState({ value: '' })
+  const [serverError, setServerError] = useState()
+  const [phrase, setPhrase] = useState()
+  const [signingIn, setSigningIn] = useState(false)
+  const [loading, setLoading] = useState(false)
+  const [tokenType, setTokenType] = useState('EMAIL_CODE')
+  const [showErrors, setShowErrors] = useState(false)
+
+  const handleEmail = (value, shouldValidate) => {
+    setEmail({
+      ...email,
+      value,
+      error: (
+        (value.trim().length <= 0 && t('Marketing/Trial/email/error/empty')) ||
+        (!isEmail(value) && t('Marketing/Trial/email/error/invalid'))
+      ),
+      dirty: shouldValidate
+    })
+  }
+
+  const requestAccess = e => {
+    e.preventDefault && e.preventDefault()
+
+    setLoading(true)
+    setServerError()
+
+    if (!me) {
+      handleEmail(email.value, true)
+
+      if (errorMessages.length > 0) {
+        setLoading(false)
+        return setShowErrors(true)
+      }
+
+      return props.signIn(
+        email.value,
+        'trial',
+        consents,
+        tokenType
+      )
+        .then(({ data: { signIn } }) => {
+          setTokenType(signIn.tokenType)
+          setPhrase(signIn.phrase)
+
+          setLoading(false)
+          setSigningIn(true)
+        })
+        .catch(error => {
+          setServerError(error)
+          setLoading(false)
+        })
+    }
+
+    setSigningIn(false)
+
+    props.requestAccess()
+      .then(() => toOnboarding())
+      .catch(error => {
+        setServerError(error)
+        setLoading(false)
+      })
+  }
+
+  const reset = () => {
+    setLoading(false)
+    setSigningIn(false)
+  }
+
+  const errorMessages = [email.error]
+    .concat(getConsentsError(t, REQUIRED_CONSENTS, consents))
+    .filter(Boolean)
+
+  return (
+    <Fragment>
+      <H1>Danke für Ihre Neugier!</H1>
+      <P style={{ marginTop: 40 }}>
+        Bitte tragen Sie unten Ihre E-Mail-Adresse ein. Im Gegenzug laden wir Sie auf einen Streifzug
```

Comment:
Hard-coded string should be moved to translations.

Reply:
Yes, strings ought to be replaced with translations keys once settle on text.